corehq/apps/domain/views/internal.py

```diff
@@ -97,8 +97,8 @@ def dispatch(self, request, *args, **kwargs):
     def internal_settings_form(self):
         can_edit_eula = toggles.CAN_EDIT_EULA.enabled(self.request.couch_user.username)
         if self.request.method == 'POST':
-            return DomainInternalForm(self.request.domain, can_edit_eula,
-                                      self.request.POST, user=self.request.user)
+            return DomainInternalForm(self.request.domain, can_edit_eula, self.request.user,
+                                      self.request.POST)
```

Comment:
I think `user` still has to be passed as a keyword argument, and as is it will raise `TypeError`.